src/VideoStorageService.py

```diff
@@ -37,7 +37,7 @@ def store_video(self, item: VideoItem) -> None:
 
     def move_video(self, cur_loc, new_name):
         new_path = os.path.join(self.storage_dir, new_name)
-        os.system(f"mv {cur_loc}* {new_path}")
+        os.rename(cur_loc, new_path)
```

Comment:
Does this remove the effect of the asterisk, because if not the two spots that this is used in needs to add the extension

Reply:
Doing some research it appears this does remove the effects of the asterisk, I'll make a suggested change to account for this

Reply:
It does indeed remove the asterisk... But why do we need the asterisk?

Reply:
I see that it's being used when 1) downloading a video, and 2) moving the storage directory. The second one does it via a loop for all files in storage. When is moving more than 1 file using this function expected?

Reply:
I'm not sure the file names include the extension, which I'm worried will cause a problem.

Reply:
My bad, the extensions are already being properly handled.